assets/apps/0000_70_dns_01-daemonset.yaml

```diff
@@ -4,15 +4,9 @@ metadata:
   labels:
     dns.operator.openshift.io/owning-dns: default
   name: dns-default
-  namespace: openshift-dns    
+  namespace: openshift-dns
 spec:
-  selector:
```

Comment:
Where's this pulling from? , do we really want to start the dns on all nodes?
Not a problem though since we are singlenode for the time being.

Reply:
To stay aligned with OpenShift, we're taking the manifest the OpenShift cluster-dns-operator would deploy. For the release we're rebasing to this is:
https://github.com/openshift/cluster-dns-operator/blob/cf8be7b279499d5010894b8c9bf79acc9853be36/assets/dns/daemonset.yaml

If we have a good reason to deviate from this, e.g. to adapt to MicroShift's potentially different networking context, we'll do that, of course.